ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/proxy/workers-rhel8/provision/cucushift-installer-rehearse-vsphere-ipi-proxy-workers-rhel8-provision-chain.yaml

```diff
@@ -31,9 +31,5 @@ chain:
     default: "test"
     documentation: |-
       The user to login the system of RHEL instance
-  - name: REMOVE_RHCOS_WORKER
```

Comment:
If possible, I would like keep the lines, and set its explicit value to "yes" for easy understanding.

Reply:
sure, added it back.